cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -193,8 +193,6 @@ __device__ inline void gpuDecodeValues(
             }
             break;
         }
-      } else if (dtype == INT96) {
```

Comment:
I think decoding of int96 should remain possible, just remove the write path.

Reply:
Reverted!